src/Microsoft.ML.Transforms/RffTransform.cs

```diff
@@ -232,6 +238,25 @@ private static string TestColumnType(ColumnType type)
             return "Expected R4 or vector of R4 with known size";
         }
 
+        /// <summary>
+        /// Convenience constructor for public facing API.
+        /// </summary>
+        /// <param name="env">Host Environment.</param>
+        /// <param name="input">Input <see cref="IDataView"/>. This is the output from previous transform or loader.</param>
+        /// <param name="name">Name of the output column.</param>
+        /// <param name="source">Name of the column to be transformed. If this is null '<paramref name="name"/>' will be used.</param>
+        /// <param name="newDim">The number of random Fourier features to create.</param>
+        /// <param name="useSin">create two features for every random Fourier frequency? (one for cos and one for sin).</param>
```

Comment:
>create two features for every random Fourier frequency? [](start = 33, length = 55)

So, if `newDim` was 10, and I set this to `true`, does the output become 20 or something?